### ISSUE TITLE
Crosswalk to MobilityDatabase data

### ISSUE THREAD

**Issue by e-lo** [cal-itp/data-infra#13]
Ability to join information from the MobilityDatabase to information in CT Data Warehouse.

MobilityDatabase.org should be the canonical source of things like:
1. Transit providers
2. Transit route names
3. Relationships between Transit Providers and GTFS feeds
4. Relationships between stop_id in a GTFS feeds and canonical station/stop names (aka shared identifiers / aliases)
5. Relationships between GTFS feeds such as transfer fares.

Mobility Database is implemented using WikiBase and managed by MobilityData.

**Comment by mcplanner** [comment]
@e-lo can you say more about this? Is this just a matter of picking the right agency IDs?

**Comment by atvaccaro** [comment]
@lauriemerrell this is done right?

**Comment by atvaccaro** [comment]
ah I'm mixing up airtable transit database and the mobility database CSV

**Comment by lauriemerrell** [comment]
Yeah no this is separate. However I think that this also might be obsolete, could check with @e-lo because MD has changed their product direction on the database significantly.
